tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -964,30 +964,30 @@ _setup_pipeline (TestOption &option)
     /** video stream with tensor_aggregator */
     str_pipeline = g_strdup_printf (
         "videotestsrc num-buffers=%d ! videoconvert ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)%lu/1 ! "
-        "tensor_converter ! tensor_aggregator frames-out=10 frames-flush=5 frames-dim=1 ! tensor_sink name=test_sink",
+        "tensor_converter ! other/tensor ! tensor_aggregator frames-out=10 frames-flush=5 frames-dim=1 ! other/tensor ! tensor_sink name=test_sink",
```

Comment:
Why do you need to specify these? or are you trying to test "obsolete" ```other/tensor``` type, explicitly?

Reply:
@myungjoo Right, for the negotiation test of old mimetype.

Reply:
Then, we need to add new testcases, copying these modified four cases.
1. the original cases (without specifying MIME types)
2. the backward compatibility checking cases (other/tensor)
3. the new type checking cases (other/tensors2)
4. mixed cases

Reply:
Yes, so I updated these testcases. Current aggregator without caps-filter will negotiate with mimetype other/tensors.
only old type : `other/tensor ! aggregator ! other/tensor`
mixed type : `other/tensors ! aggregator ! other/tensor` and `other/tensor ! aggregator ! other/tensors`
current mimetype : `other/tensors ! aggregator ! other/tensors`

Reply:
Ah.. it's just you didn't do cartesian product on the test cases, but mixed in different test-dimensions. It's fine, then.